aten/src/ATen/native/Splitting.cpp

```diff
@@ -2,7 +2,9 @@
 #include <ATen/TensorIterator.h>
 #include <ATen/core/PhiloxRNGEngine.h>
 #include <ATen/CPUGeneratorImpl.h>
+#ifdef USE_CUDA
```

Comment:
Nuh uh, you're not allowed to do this. Put the CUDA code in native/cuda/Splitting.cpp

Reply:
Ohhh, you are right. This Macro is never defined here.